fairlearn/postprocessing/_postprocessing.py

```diff
@@ -12,14 +12,16 @@
 class PostProcessing:
     """Abstract base class for postprocessing approaches for disparity mitigation.
 
-    :param unconstrained_predictor: a predictor with a `predict` function that has already been
-        trained on the training data; the predictor will subsequently be used in the mitigator
-        for unconstrained predictions; can only be specified if `estimator` is None
+    :param unconstrained_predictor: A predictor with a :code:`predict(X)` method that has already
```

Comment:
can we document what style we're following somewhere, maybe in some section of CONTRIBUTING.md?

Reply:
For the most part here, I was just making the text (roughly) the same. We use standard autodoc comments